operator-pipeline-images/tests/entrypoints/test_bundle_dockerfile.py

```diff
@@ -0,0 +1,20 @@
+from unittest.mock import MagicMock, patch
+
+from operatorcert.entrypoints.bundle_dockerfile import generate_dockerfile_content
+
+
+@patch("operatorcert.entrypoints.bundle_dockerfile.setup_argparser")
```

Comment:
`setup_argparser` is not called in the function so the mock is not needed.

Reply:
Removed. I forgot to remove it before.